### PR TITLE
:sparkles: use Jinja for description_key and other fields

### PR DESCRIPTION
**Pull request by Marigold**

### CODE REVIEW

File: etl/grapher_helpers.py [hunk 1]
```diff
@@ -174,6 +174,8 @@ def _yield_wide_table(
             for k in (
                 "description",
                 "description_short",
+                "description_from_producer",
+                "description_processing",
```
Comment (lucasrodes) [inline etl/grapher_helpers.py:178]: any chance that `description_key` is missing here?

**Review by lucasrodes — COMMENTED** [review]
LGTM ー but support for `description_key` might be missing?

File: etl/grapher_helpers.py (hunk 1 above)
Comment (Marigold) [reply etl/grapher_helpers.py:178]: `description_key` is a list which needs a special treatement and is right below.

Comment (lucasrodes) [reply etl/grapher_helpers.py:178]: ah, my bad ー I scanned rapidly, and didn't see it

**Review by lucasrodes — APPROVED** [review]
LGTM! Thanks for doing this

**Pull request merged**